src/olympia/amo/tasks.py

```diff
@@ -28,16 +28,14 @@ def send_email(recipient, subject, message, from_email=None,
     if html_message:
         result.attach_alternative(html_message, 'text/html')
     try:
-        result.send(fail_silently=False)
+        result.send()
         return True
     except Exception as e:
-        log.error('send_mail failed with error: %s' % e)
-        if async:
+        if max_retries:
```

Comment:
hmm. Doesn't `max_retries == None` mean use the default, rather than no retries?

Reply:
@EnTeQuAk correct me if I'm wrong, but the default should be no retries - we have to explicitly trigger the retry for it to happen.

Reply:
Oh, actually: http://docs.celeryproject.org/en/latest/userguide/tasks.html#Task.max_retries

> Note
> You have to call retry() manually, as it won’t automatically retry on exception..
> The default is 3. A value of None will disable the retry limit and the task will retry forever until it succeeds.

I probably need to remove that `if`, always calling `.retry()`, and let celery obey `max_retries` by itself